marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/triggers.core.artifact/createArtifactTrigger.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ml-dh-entity-create",
```

Comment:
The name property here will conflict with the existing trigger for entities

Reply:
Thanks! I already change it.